src/gpsea/preprocessing/_config.py

```diff
@@ -455,7 +455,7 @@ def load_phenopackets(
     # Turn phenopackets into a cohort using the cohort creator.
     # Keep track of the progress by wrapping the list of phenopackets
     # with TQDM 😎
-    cohort_iter = tqdm(phenopackets, desc="Patients Created", file=sys.stdout)
+    cohort_iter = tqdm(phenopackets, desc="Patients Created", file=sys.stdout, unit="files")
```

Comment:
As far as I can see, this still says `"files"` instead of individuals, and the `desc` includes "Patients".

Reply:
Ooof, yah I'm sorry I thought I had pushed that change when I made the comment before. It should be fixed now!